Localize/lcl/de/src/TestFramework/MSTest.Core/Resources/xlf/FrameworkMessages.xlf.lcl

```diff
@@ -0,0 +1,506 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<LCX SchemaVersion="6.0" Name="FrameworkMessages.xlf" PsrId="308" FileType="1" SrcCul="en-US" TgtCul="de-DE" xmlns="http://schemas.microsoft.com/locstudio/2006/6/lcx">
+  <Props>
+    <Str Name="BranchName" Val="" />
```

Comment:
Should we fill these values?

Reply:
We shouldn't touch the `*.lcl` files as they're owned by the localization team who's responsible for the actual translations.